src/bundle/ui-dev/src/modules/content-tree/components/popup-actions/popup.actions.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect, useRef } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import Icon from '../../../common/icon/icon';
+
+const PopupActions = ({ listRef, options }) => {
+    const containerRef = useRef();
+    const containerItemsRef = useRef();
+    const [isExpanded, setIsExpanded] = useState(false);
+    const toggleExpanded = () => {
+        setIsExpanded((prevState) => !prevState);
+    };
+    const getHeaderActions = () => {
```

Comment:
Is it identical to `getPrefixActions` from `ListItemComponent`?

Reply:
It takes data from different property